app/controllers/LilaController.scala

```diff
@@ -43,7 +43,7 @@ abstract private[controllers] class LilaController(val env: Env)
   protected val keyPages       = new KeyPages(env)
   protected val renderNotFound = keyPages.notFound _
 
-  implicit protected def LilaFunitToResult(@silent funit: Funit)(implicit ctx: Context): Fu[Result] =
+  implicit protected def LilaFunitToResult(@silent funit: Funit)(implicit req: RequestHeader): Fu[Result] =
     negotiate(
```

Comment:
Somehow this completely unrelated change seems to be required, even when doing nothing but declaring `protected def EnableSharedArrayBuffer` below.

Reply:
Yeah it's a compiler glitch with implicit resolution. The change is fine